websites/S/statcord.com/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+  clientId: "692810788196581376"
+}),
+
+ browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: browsingStamp
+  };
+
+  if (document.location.hostname === "statcord.com") {
+    presenceData.details = "Viewing Page:";
+    presenceData.state = "Viewing At Home Page";
+    if (document.location.pathname.includes("/bot/")) {
+      presenceData.details = "Viewing:";
+      presenceData.state = `${document.querySelector("#overview > div:nth-child(1) > div > div > div > div.col.mr-2 > div.h5.mb-0.font-weight-bold").textContent} Statistics`;
+    } else if (document.location.pathname.includes("/login/")) {
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = `Login`;
+    } else if (document.location.pathname.includes("/profile")) {
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = "Bots List";
+    } else if (document.location.pathname.includes("/add")) {
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = `Add Bot`;
+    }
+  }
+
+  if (document.location.hostname === "docs.statcord.com") 
+    presenceData.details = "Viewing Docs";
```

Comment:
Support docs better

Reply:
I did try with checking css and even geting location of mouse but still cant do that

Reply:
+ i tryed using there `?id=quick-start` the think called id but did not work how i wanted did not check for it

Reply:
document.location.hash